src/main/java/utilties/models/Game.java

```diff
@@ -50,8 +55,11 @@ private void initializeGame(){
      * @return Updated Map
      */
     public Map nextTurn(){
-        PlayerFunctions.Move(map, new Location(player.getLocation().getX() + 1, player.getLocation().getY()), player);
-        PlayerFunctions.Move(map, new Location(opponent.getLocation().getX() - 1, opponent.getLocation().getY()), opponent);
+
+        // NOTE this is just an example of how the interpreter will work - using dummy values for now
+        playerCommands.get(0).performAction(map, new Location(player.getLocation().getX() + 1, player.getLocation().getY()), player);
+        opponentCommands.get(0).performAction(map, new Location(opponent.getLocation().getX() - 1, opponent.getLocation().getY()), opponent);
```

Comment:
I really like this implementation.  Much more streamlined model than before.  Great idea.